### DISCUSSION TITLE
Homepage routing

### DISCUSSION THREAD

**Discussion by Max2983** [requarks/wiki#5242] [Help / Questions]
I have installed my Wiki.JS at http://146.190.230.117 (I plan to add a domain name later). Also I have created and published two pages. I want my wiki being public, so any visitor could read articles. So I created a Group with rights [to read:pages, read:assets, read:comments]. But when I open http://146.190.230.117 (in a different browser or incognito mode) it shows me a standard welcome page proposing to create a homepage or administrate.
<img width="1436" alt="Screenshot 2022-05-01 at 8 09 32" src="https://user-images.githubusercontent.com/85618579/166156896-01aa5374-d29c-4686-a8a8-e77322e59ae5.png">

How do I make people being shown the page I created instead? Where do I set up this page being a homepage?
Also, why visitor is proposed to create a page when he/she supposed to belong to the group I created and that does NOT have rights to create pages?
Please help!

**Comment by NGPixel** [comment]
What are the page rules you entered in your groups?

**Reply by Max2983** [reply]
![](https://user-images.githubusercontent.com/85618579/166187854-6574acb3-d369-43b7-8258-b8a3c73e5928.png)

**Reply by Max2983** [reply]
I have found the solution.
I just needed to click that button '+ Create home page' and actually create a page. Once I did that, everything worked properly.
And getting back to question How do you define a homepage for your WikiJS... the answer is: any page that has the only word '_home_' in its path settings will be the starting page.

Sorry for bothering you. My stupid.
